src/encoded/root.py

```diff
@@ -264,26 +264,39 @@ def jsonld_type(self):
     def content(self, request):
         '''Returns -object- with pre-named sections'''
         sections_to_get = ['home.introduction']
+        user = request._auth0_authenticated if hasattr(request, '_auth0_authenticated') else True
         return_list = []
         for section_name in sections_to_get:
-            try:
-                res = request.embed('/static-sections', section_name, '@@embedded', as_user=True)
+            try: # Can be caused by 404 / Not Found during indexing
+                res = request.embed('/static-sections', section_name, '@@embedded', as_user=user)
                 return_list.append(res)
-            except:
+            except KeyError:
                 pass
         return return_list
 
-#    @calculated_property(schema={
-#        "title": "Announcements",
-#        "type": "array"
-#    })
-#    def announcements(self, request):
-#        '''Returns list of latest announcements'''
-#        try:
-#            ## Doesn't go thru permissions correctly (status:draft visible to non-logged-in-users)
```

Comment:
This comment is no longer accurate (?)
Tested and permissions work fine; perhaps is due to `category="page"` ?
Regardless, is working as intended and permissions are correctly observed, even with as_user=True instead of email.